SoftwareGuide/Latex/Introduction/Installation.tex

```diff
@@ -589,6 +632,58 @@ \section{Getting Started With ITK}
 to see the output. It is therefore preferable to run the executable from the DOS
 command line by starting the \code{cmd.exe} shell first.
 
+\section{Using ITK as an External Library}
+\label{sec:UsingITKAsExternalLibrary}
+
+For a project that uses ITK as an external library, it is recommended to
+specify the individual ITK modules in the \code{COMPONENTS} argument in the
+\code{find_package} CMake command:
+
+\small
+\begin{minted}[baselinestretch=1,fontsize=\footnotesize,linenos=false,bgcolor=ltgray]{bash}
+find_package(ITK COMPONENTS Module1 Module2 REQUIRED)
```

Comment:
Better to give concrete examples

Reply:
e.g. https://github.com/KitwareMedical/ITKMorphologicalContourInterpolation/blob/716b4a3ccbb329e54d9e2bd1d1d914699e1ccb1b/examples/CMakeLists.txt#L5-L12